assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -105,6 +112,28 @@ const DashboardIdeasWidget = ( {
 		[ setHash, setActiveTabIndex ]
 	);
 
+	// Any time the pagination value changes, scroll to the top of the container.
+	useUpdateEffect( () => {
+		const tabBarRectangle = tabBarHeaderRef.current.getBoundingClientRect();
```

Comment:
Is it possible the ref might not be set here? The ref isn't added in all cases as there is an early return below if there are zero total ideas.

Reply:
Yeah, I think it's possible, I need to protect against it 👍🏻